httperror/http_error.go

```diff
@@ -60,28 +61,28 @@ func New(ctx appcontext.AppContext, err error, instance string) ProblemDetails {
 		Type:     appError.Code(),
 		Title:    appError.Error(),
 		Detail:   appError.Detail(),
-		Status:   mapAppErrorToHttpStatusCode(appError),
+		Status:   mapAppErrorToHTTPStatusCode(*appError),
 		Instance: instance,
 		TraceID:  ctx.TraceID(),
 		Errors:   appError.ValidationErrors(),
 	}
 }
 
-func mapAppErrorToHttpStatusCode(appError apperror.AppError) int {
+func mapAppErrorToHTTPStatusCode(appError app.Error) int {
```

Comment:
This should be a getter of app.Error

```go
func (e Error) HTTPStatus() int {
// ...
}
```

So here you would call it

Reply:
I don't think the app.Error should know anything about the HTTP protocol.

Reply:
No problem